docs/xmlprofiles.xml

```diff
@@ -0,0 +1,1097 @@
+<xml>
```

Comment:
This file is not referenced in the docs. Should it be removed?

Reply:
It was left as the reference for the examples. It was used before the migration to the XMLTester.xml file. It can be removed in any case.

Reply:
I'll remove it then... done.